daemon/graphdriver/graphtest/graphbench_unix.go

```diff
@@ -172,10 +172,10 @@ func DriverBenchDiffApplyN(b *testing.B, fileCount int, drivername string, drive
 		b.StopTimer()
 		arch.Close()
 
-		if applyDiffSize != diffSize {
-			// TODO: enforce this
-			//b.Fatalf("Apply diff size different, got %d, expected %s", applyDiffSize, diffSize)
-		}
+		// TODO: enforce this
+		//if applyDiffSize != diffSize {
```

Comment:
This removes one linter warning, but introduces another:
```
daemon/graphdriver/graphtest/graphbench_unix.go:167:3: applyDiffSize declared but not used (typecheck)
		applyDiffSize, err := driver.ApplyDiff(diff, "", arch)
		^
```

I guess the best way would be to use `// lint:ignore SA9003 because TODO` instead of commenting out the check.